stdlib/REPL/src/REPL.jl

```diff
@@ -1190,10 +1190,11 @@ function setup_interface(
                 pkgid = Base.PkgId(Base.UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "Pkg")
                 if Base.locate_package(pkgid) !== nothing # Only try load Pkg if we can find it
                     Pkg = Base.require(pkgid)
+                    REPLExt = get_extension(Pkg, :REPLExt)
```

Comment:
```suggestion
                    REPLExt = Base.get_extension(Pkg, :REPLExt)
```

or?

Reply:
Yeah. I'll merge this once the other issue is resolved as build will fail